### ISSUE TITLE
Sampling of TimeSeries

### ISSUE THREAD

**Issue by davidenke** [esnet/pond#216]
Hello,

is it possible to introduce some kind of sampling to visualize large data sets? We have to show a time series with data points (3 to 5) in a 15 minute interval for about a year which can be zoomed in to a single day. So basically about 35000 data points with 3 to 5 values each.

I've seen some interpolation methods in the `TimeSeries` class and wanted to implement some kind of sampling myself without success. My idea was to keep the initial `TimeSeries` and to provide a specific one for the view with sampled data dependent on the current `TimeRange`. Using `quantile` helps to sample the data, but I'm losing the timestamp to create a new `TimeSeries` as it returns only the values.

In short: is there a way to implement some kind of sampling to allow visualizing large data sets?
